pkg/command/resolve.go

```diff
@@ -160,8 +160,11 @@ func (opts *ResolveOptions) Execute(cmd *cobra.Command, args []string) error {
 		return errors.New("'im bundle' does not take any arguments")
 	}
 
+	if opts.Ctx == nil {
+		opts.Ctx = signals.NewContext()
+	}
```

Comment:
I'm curious what your entrypoint is for unit testing 🤔 

I ask because I'd generally prefer to call `ExecuteContext(ctx)` and use `cmd.Context()` here, but chmoel's fork of cobra for Tekton didn't have this last I checked.  I'll check again, and maybe we can poke him about updating his fork.

See: https://godoc.org/github.com/spf13/cobra#Command.Context

Reply:
I meant to exclude the ctx change from this PR and just get namespace/clients sorted first and then bring up the entry point / testing side of commands in mink in a future PR. Let me remove the ctx change for now and we can play with that on another PR